### DISCUSSION TITLE
How to change theme?

### DISCUSSION THREAD

**Discussion by turbanisch** [sbarex/QLMarkdown#128] [General]
Sorry for this blunt question - but how can I change the theme or define a new one? The dropdown menu in QLmarkdown's preferences gives me some options but no themes seem to be pre-installed and I can't find an example of how the CSS file would have to look like.

In my case, I am using QLmarkdown to render files as source code and would like to harmonize the theme with what I am using with the SourceCodeSyntaxHighlight extension.

**Comment by setanarut** [comment]
You can download the default css theme file by holding down the alt (option) key, the whole theme is controlled with this file. There is also an import option in the same menu.

![Ekran Resmi 2024-04-09 00 33 02](https://github.com/sbarex/QLMarkdown/assets/36481442/bfe56273-b202-41ab-8b23-a97961efb840)
